src/instructions/tables.rs

```diff
@@ -79,6 +96,20 @@ pub fn sidt() -> DescriptorTablePointer {
     idt
 }
 
+/// Get the address of the current LIDT.
+#[inline]
+pub fn sldt() -> DescriptorTablePointer {
+    let mut sldt: DescriptorTablePointer = DescriptorTablePointer {
+        limit: 0,
+        base: VirtAddr::new(0),
+    };
+    unsafe {
+        #[cfg(feature = "inline_asm")]
+        asm!("sldt [{}]", in(reg) &mut sldt, options(nostack, preserves_flags));
+    }
+    sldt
+}
+
```

Comment:
I assume you mean the [local descriptor table](https://wiki.osdev.org/LDT) here? Is there any use case for this table in 64-bit mode? Otherwise, I see no reason to add it.

Reply:
There is no real use case, so yeah I can remove it~